### PR TITLE
Initialize express app

### PR DESCRIPTION
**Pull request by fimion**
a basic express app

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var express = require('express');
```
Comment (mtliendo) [inline app.js:1]: I think we should forego the use of `var` in favor of `const` and `let`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "devlifts-api",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
```
Comment (mtliendo) [inline package.json:6]: Instead of jumpings off via a `.bin` exec, can we just put this in `index.js`?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<html>
```
Comment (mtliendo) [inline public/index.html:1]: Was this made this express-generator?

**Review by mtliendo — APPROVED** [review]
I didn't see any requirement for this to _not_ be made with `express generator`, so in that case, I would approve, and the `var` stuff can be made in another PR as cleanup, unless you wanted to push up the changes.  

Thanks for your contribution!

**Review by jchiatt — APPROVED** [review]

**Pull request merged**